### DISCUSSION TITLE
v0.3.10

### DISCUSSION THREAD

**Discussion by stefansundin** [stefansundin/privatkopiera#179] [Announcements]
- Fixade nrk.no.
- Fixade dr.dk.

**Full Changelog**: https://github.com/stefansundin/privatkopiera/compare/v0.3.9...v0.3.10

<hr /><em>This discussion was created from the release <a href='https://github.com/stefansundin/privatkopiera/releases/tag/v0.3.10'>v0.3.10</a>.</em>

**Comment by rogerrapp** [comment]
TV4-Play har funkat ett bra tag, men inte längre.
Står bara Loading....
Ubuntu 20.04 LTS  Brave [Version 1.33.106 Chromium: 96.0.4664.110 (Official Build) (64-bit)](https://brave.com/latest/)
Samma fel från Chromium.
Jag kan tyvärr inte säkert sätta det i samband med 3.10. TV4 kan mycket väl hittat på något nytt jävulskap.

**Reply by stefansundin** [reply]
Diskussion i #185.

**Comment by simon2874** [comment]
Jag har försökt ladda ner ett avsnitt av 'Efter fem' som finns på TV4play men det fungerar inte. Jag använder Firefox och Windows 11. Går det att fixa?

**Reply by stefansundin** [reply]
Se diskussionen i https://github.com/stefansundin/privatkopiera/issues/185.

**Comment by Tofta1386** [comment]
Sedan 1 oktober får man ingen svensk text längre från Svtplay filmer. Finns det någon lösning på detta?

**Comment by Mogge555** [comment]
Får inte med svenska texter eller subtitles från utlländska program från svt play för över en vecka tillbaka i tiden

**Reply by Tofta1386** [reply]
Jag har provat med framgång att använda svtplay-dl enligt exempel nedan och mixat alla filerna till en .mkv fil med MKV toolnix

svtplay-dl -S https://www.svtplay.se/video/KM66BDD/once-upon-a-time-in-america?id=KM66BDD

**Reply by Mogge555** [reply]
Jättegrattis att du har lyckats men det måste vara ett jättejobb för att mixa för en endaste film eller program det verkar som ingen bryr sig om privatkopiera mera går väl i graven snart

**Reply by stefansundin** [reply]
> det verkar som ingen bryr sig om privatkopiera mera går väl i graven snart

Det är ingen som tvingar dig att använda Privatkopiera. Du har väl gjort precis noll för att hjälpa till? Om du ska klaga så kan du gå någon annanstans.

**Reply by Phiplex** [reply]
Stefan, vill att du ska veta att det finns många, och jag är en av dem, som mycket uppskattar allt det arbete du gör frivilligt för att underhålla Privatkopiera. Jag använder helst Privatkopiera, då jag upplever det som enklare efter att tidigare även testat svtplay-dl (som visserligen också är bra och fungerar utmärkt, så ingen negativ kritik mot det heller). Är därför mycket tacksam så länge du orkar underhålla Privatkopiera. Så strunta helt i den oberättigade negativ kritik som vissa ger uttryck för utan anledning. ”Keep up your good work” 👍🏻

**Reply by Mogge555** [reply]
jag har inte klagat jag har upplevt att det varit bra och enkelt och haft tålamod när man gör om på play kanaler men jag har hittat andra vägar att ta hem det jag vill ha men jag vill tacka dig stefan för ditt innerliga suveräna arbete med privatkopiera hälsningar Mogge 555

**Reply by Mogge555** [reply]
stefan sundin du ju ju skrivit att du inte bryr  dig om det så mycket längre och det märks sedan skiter jag vad du tycker om mig jag är inte en data programerare jag är en vanlig enkel person med goda erfarenheter jag skickar ett stort tack till dig men någonting är fel hardsub  fungerar  inte  längre svt  tv4 är  fungerar inte vtt filen går inte göra om srt för liten fil jag vet att det inte är ditt fel.Jag vet vad du sa ingen tvingar dig att vara här men jag tänker på alla andra men jag har gått andra vägar med andra program någon säger att svt har ändrat från playen från hardsub till vanlig sub från sin egen hemsida så lär det alldrig funka sedan hoppas jag inte på ett argt svar från dig. Jag är villig att ge dig ett ett bidrag om du löser här  mycket klurigt

**Comment by Mogge555** [comment]
Ni som sitter här och inte hittar svenska undertexter jag kanske har en lösning till er det finns ett program som heter Subtitle Edit med det programmet kan bolla lite hit och dit blir engelska först sedan kan subba till svenska allting måste läggas i samma mapp funkar till 99%

**Comment by peppe5151** [comment]
Får inga undertexter längre! Finns det nån lösning på det?

**Reply by Bub0** [reply]
Testa gör på rätt sätt!

**Reply by peppe5151** [reply]
har gjort som jag gjort förut!
En Instruktion på rätt sätt är välkommet!

**Reply by Bub0** [reply]
Testa berätta vad du gjort och länka med!
Alltså en riktig problemrapport och inte bara det funkar inte!
Ibland tar man sig för pannan. 
Gör det manuellt själv annars, men det funkar...

**Reply by peppe5151** [reply]
Jag kopierar genererad länk från privatkopiera och klistrar in i kommandotolken i ffmpeg.

**Reply by stefansundin** [reply]
Du kan vara drabbad av problemet som är beskrivet i #120. Det finns en workaround.

**Comment by dibblah** [comment]
Hej Stefan!

Installerade och fick Privatkopiera att fungera finfint på min Mac Mini 2 :)

Men undrar om man kan ändra var det sparar den färdiga videofilen?

På Mac sparas den nämligen i **Hem** (roten av användarbiblioteket), hade varit smidigt om man kunde välja **Hämtade Filer** eller ännu bättre valfri folder.

Med vänlig hälsning, dibblah

**Reply by stefansundin** [reply]
Hej hej. Det kommer en feature som hjälper med detta i v0.4.0 (95dfdc64a3e5e4ed64b541395efba04216da80cc). Du kommer kunna konfigurera en sökväg där filerna ska sparas, som då inkluderas automatiskt i ffmpeg-kommandot som genereras.

Normalt sett så har jag bara sagt att man ska navigera kommandotolken till mappen där man vill att filerna ska sparas (t.ex. med `cd Downloads`), men för nybörjare så är inte självklart hur det här fungerar.

Notera att Privatkopiera inte kan verifiera att sökvägen som man skriver in är korrekt, så dubbelkolla att kommandot som genereras ser korrekt ut. Mappen måste också redan vara skapad.
